<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Math TeX Gyre</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>LOT_ID:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 226, 52);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="value">
    <number>16</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="readbutton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(164, 0, 0);</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="writebutton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>write</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_DI1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>DI1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_DI2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>DI2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_DI3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>DI3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_DI4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>DI4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_DO4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>DO4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_DO2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>DO2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_DO1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>DO1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_DO3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>DO3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readbutton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>read</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
